traffic_ops/testing/api/v4/servicecategories_test.go

```diff
@@ -104,6 +112,34 @@ func CreateTestServiceCategories(t *testing.T) {
 	}
 }
 
+func CreateTestServiceCategoriesAlreadyExist(t *testing.T) {
+	firstServiceCategory := tc.ServiceCategory{}
+	if len(testData.ServiceCategories) > 0 {
+		firstServiceCategory = testData.ServiceCategories[0]
+	} else {
+		t.Fatalf("cannot CREATE DUPLICATE Service Category, test data does not have service categories")
+	}
+	resp, reqInf, err := TOSession.CreateServiceCategory(firstServiceCategory, client.RequestOptions{})
+	if err == nil {
+		t.Errorf("Expected service_category name already exists. but got: %v - alerts: %+v", err, resp.Alerts)
+	}
+	if reqInf.StatusCode != http.StatusBadRequest {
+		t.Fatalf("Expected 400 status code, got %v", reqInf.StatusCode)
+	}
+}
+
+func CreateTestServiceCategoriesInvalidName(t *testing.T) {
+	firstServiceCategory := tc.ServiceCategory{}
+	firstServiceCategory.Name = ""
+	resp, reqInf, err := TOSession.CreateServiceCategory(firstServiceCategory, client.RequestOptions{})
+	if err == nil {
+		t.Errorf("Expected service_category name already exists. but got: %v - alerts: %+v", err, resp.Alerts)
```

Comment:
I don't believe the expectation is that the service category name already exists in this test

Reply:
updated error message.